lib/minitest/event_prof_formatter.rb

```diff
@@ -42,14 +44,20 @@ def by_groups(profiler)
         profiler.results[:groups].each do |group|
           description = group[:id][:name]
           location = group[:id][:location]
+          time = group[:time]
+          time_percentage = time_percentage(time, profiler.total_test_time)
```

Comment:
Here we want to show the percentage of event time to the total **group time** (not the whole suite time).

So, we need to track each example time too.

Reply:
Quick question, sometimes the example total time can be equal to the total_test_time?